spec/support/capybara.rb

```diff
@@ -14,4 +14,4 @@
     desired_capabilities: capabilities
 end
 
-Capybara.javascript_driver = :headless_chrome
+Capybara.javascript_driver = :chrome
```

Comment:
Have you updated to Chrome 59? If so, I think this shouldn't be necessary. If you're still getting that error after updating, then certainly leave it.